spec/Section 3 -- Type System.md

```diff
@@ -341,9 +341,17 @@ client-specific primitive for time. Another example of a potentially useful
 custom scalar is `Url`, which serializes as a string, but is guaranteed by
 the server to be a valid URL.
 
+When defining an additional scalar, GraphQL systems should use the
+`@specifiedBy` directive to provide a URL pointing to a human-readable
+specification of the data format, serialization, and coercion rules for the
+scalar. For example, a GraphQL system providing a `UUID` scalar might link to
+RFC 4122, or some document defining a reasonable subset of that RFC. If a
+specification URL is present, systems must conform to the described rules.
+Built-in scalar types should not provide a URL.
+
 ```graphql example
-scalar Time
-scalar Url
+scalar UUID @specifiedBy(url: "https://tools.ietf.org/html/rfc4122")
+scalar URL @specifiedBy(url: "https://tools.ietf.org/html/rfc3986")
 ```
```

Comment:
It would be great to elaborate a bit here about how we expect client tooling to use this to provide additional behavior, and maybe even a note about how two GraphQL services with different named scalars specified by the same url should be imbued with the same behavior and conversely two services might have a scalar by the same name but specified with different urls.

Reply:
Hmm, that's a good idea. Since I believe this is basically ready to go as-is (once the implementation lands, which should be soon?) I might put that content into a second PR to avoid holding this one up over additional wording quibbles.